app/controllers/tag_controller.rb

```diff
@@ -534,13 +534,18 @@ def stats
     @start = params[:start] ? Time.parse(params[:start].to_s) : Time.now - 1.year
     @end = params[:end] ? Time.parse(params[:end].to_s) : Time.now
     tagname = params[:id]
-
+    
+    @tag_name = params[:id]
     @tags = Tag.where(name: params[:id])
-    @tag_notes = @tags.first.contribution_graph_making('note', @start, @end)
-    @tag_wikis = @tags.first.contribution_graph_making('page', @start, @end)
-    @tag_questions = @tags.first.quiz_graph(@start, @end)
-    @tag_comments = @tags.first.comment_graph(@start, @end)
-    @subscriptions = @tags.first.subscription_graph(@start, @end)
+
+    unless @tags.empty? 
```

Comment:
Hey @chen-robert, thanks for opening this :+1: almost there..just one little improvement that we could add to stop code climate from complaining...lets change this to `return if @tags.empty`